### PR TITLE
Reader Stream Refresh: add follow button to post card

### PR DESCRIPTION
**Pull request by bluefuton**
Add a follow button to the top right of the new stream cards in Reader, except on Following, team stream and Discover.

Fixes https://github.com/Automattic/wp-calypso/issues/8526.

5fcad452-8bd6-11e6-86f0-22b84722b5b7
### todo
- [x] Add button component
- [x] Hide from Following stream
- [x] Position and style button (@jancavan)

### CODE REVIEW

**Comment by bluefuton** [comment]
I've hidden the follow button for Following and also the team stream.

**Comment by jancavan** [comment]
Added styles and corrected Follow buttons positioning in: https://github.com/Automattic/wp-calypso/pull/8893/commits/a8a9069bf2f4163d53452749cc60cc87852345ad

**Before:**
screenshot 2016-10-31 14 37 20

**After:**
screenshot 2016-10-31 14 37 58

**Comment by jancavan** [comment]
- [ ] On Discover, there needs to only be one Follow button (on the header cake)

File: client/blocks/reader-post-card/index.jsx [hunk 1]
```diff
@@ -82,7 +84,7 @@ export default class RefreshPostCard extends React.Component {
 	}
 
 	render() {
-		const { post, site, feed, onCommentClick } = this.props;
+		const { post, site, feed, onCommentClick, showPrimaryFollowButton } = this.props;
```
Comment (samouri) [inline client/blocks/reader-post-card/index.jsx]: is there a non-primary follow button?

Comment (bluefuton) [inline client/blocks/reader-post-card/index.jsx]: Yes! In the post options menu:

screen shot 2016-11-02 at 12 16 36

I thought `showFollowButton` set to `false` might indicate that it was hidden everywhere on the card.

**Comment by samouri** [comment]
Code looks good to me.  Checking functionality now…

**Comment by samouri** [comment]
Things tested: 
- [x] Safari: Follow button exists and looks nice while resizing. Does not exist in Following/Search/Discover
- [x] Chrome: same as safari

🚢

**Pull request merged**